CATS/DB.pm

```diff
@@ -40,29 +40,8 @@ sub object_by_id
     select_object($table, { id => $id });
 }
 
-sub new_id
-{
-    unless ($dbh) {
-        open my $fnext_id, '<', 'judge_next_id.txt';
-        my $id = <$fnext_id>;
-        chomp $id;
-        close $fnext_id;
-        open $fnext_id, '>', 'judge_next_id.txt';
-        printf $fnext_id "%d\n", $id + 1;
-        return $id;
-    }
-    if ($CATS::Config::db_dsn =~ /Firebird/)
-    {
-        $dbh->selectrow_array(q~SELECT GEN_ID(key_seq, 1) FROM RDB$DATABASE~);
-    }
-    elsif ($CATS::Config::db_dsn =~ /Oracle/)
-    {
-        $dbh->selectrow_array(q~SELECT key_seq.nextval FROM DUAL~);
-    }
-    else
-    {
-        die 'Error in new_id';
-    }
+sub new_id {
+    Digest::MD5::md5_hex(Encode::encode_utf8($_[1]));
```

Comment:
This will totally break cats-main. Id generation must still work with the database.